packages/caip/src/adapters/banxa/index.ts

```diff
@@ -55,11 +71,15 @@ export const getSupportedBanxaAssets = () =>
  * map ChainIds to Banxa blockchain codes (ETH, BTC, COSMOS),
  * since some Banxa assets could be on multiple chains and their default
  * chain won't be exactly the same as ours.
+ *
+ * https://docs.google.com/spreadsheets/d/1KU6J1Hl4vxBTIWCwWdrFfSadNltuFheQRoJyPrd0LMQ/edit#gid=631982242
```

Comment:
dug this link up from the original PR, couldn't find this for reference which may have solved things from the get-go